.github/workflows/tests.yml

```diff
@@ -55,4 +55,4 @@ jobs:
         run: yarn compile
 
       - name: Run tests
-        run: yarn test --network zkSyncTestNode
+        run: yarn test --network zkSyncTestNode
```

Comment:
Missed new line

Reply:
Maybe it's a github diff mistake, because i merged tests.yml from [matter-labs repo](https://github.com/matter-labs/era-permit2/blob/0x000000000022D473030F116dDEE9F6B43aC78BA3-zksync-era/.github/workflows/tests.yml). Therefore prettier doesn't applied for this file.

Reply:
Can you restore it?